pkg/monitor/dynkeepalived.go

```diff
@@ -100,6 +100,7 @@ func doesConfigChanged(curConfig, appliedConfig *config.Node) bool {
 	// approach we should avoid asymetric configuration
 	if curConfig.EnableUnicast {
 		if os.Getenv("IS_BOOTSTRAP") == "no" && len(curConfig.LBConfig.Backends) < 2 {
+			log.Warnf("Invalid keepalived.conf - number of api backends must be at least 2 got %d", len(curConfig.LBConfig.Backends))
```

Comment:
nit: this function technically is not specific to API, it can be as well used for any arbitrary type of backend; let's keep the error message generic, scratch the word "api"